joomla/joomla-cms
# [4.0] [com_users] Move JS to separate file + styling tweaks

Pull Request for Issue # .

### Summary of Changes

- Move the inline JS to separate files
- Rewrite the jQuery in Vanilla JS
- Some styling tweaks to the TFA settings on the `user` view

### Test 1

- Deleting a Usergroup that contains a user provides you 2 confirmations


### Test 2

- Enable a Two Factor Authentication plugin
- In the backend, go to edit a profile, then navigate to the `Two Factor Authentication` tab.
- Select an option from the `Authentication Method` dropdown

This should display the steps to setup TFA
